tower-timeout/Cargo.toml

```diff
@@ -8,21 +8,21 @@ name = "tower-timeout"
 #   - README.md
 # - Update CHANGELOG.md.
 # - Create "v0.1.x" git tag.
-version = "0.1.1"
+version = "0.3.0-alpha.1"
 authors = ["Tower Maintainers <team@tower-rs.com>"]
 license = "MIT"
 readme = "README.md"
 repository = "https://github.com/tower-rs/tower"
 homepage = "https://github.com/tower-rs/tower"
-documentation = "https://docs.rs/tower-timeout/0.1.1"
+documentation = "https://docs.rs/tower-timeout/0.3.0-alpha.1"
 description = """
 Apply a timeout to requests, ensuring completion within a fixed time duration.
 """
 categories = ["asynchronous", "network-programming"]
 edition = "2018"
 
 [dependencies]
-futures = "0.1.26"
-tower-service = "0.2.0"
-tower-layer = "0.1.0"
-tokio-timer = "0.2.6"
+tower-service = "0.3.0-alpha.1"
+tower-layer = { version = "0.3.0-alpha.1", path = "../tower-layer" }
+tokio-timer = "0.3.0-alpha.1"
+pin-project = "0.4.0-alpha.10"
```

Comment:
do we want to make sure the whole project is using the same version?

Reply:
Sure -- I'll do that in a follow-up PR.